common/helpers.py

```diff
@@ -355,3 +356,23 @@ def get_hostname():
         return open(path, 'r').readlines()
 
     return []
+
+
+def grep(pattern, files):
```

Comment:
Can we use common.searchtools? e.g. https://github.com/dosaboy/hotsos/blob/master/plugins/openstack/09agent_errors since it supports (a) parallel searches (b) gzipped files and (c) tagged results to make it much easier to identify positive hits.

Reply:
done.